app/Http/Controllers/ProductController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\common;
```

Comment:
here you are violating PSR-1, read [PSR-1](https://www.php-fig.org/psr/psr-1/#3-namespace-and-class-names)

here is a part of it `Class names MUST be declared in StudlyCaps` & `The term "class" refers to all classes, interfaces, and traits.`

these standards are very important, do you know that laravel depend on one of these?